diff --git a/node_modules/react-native-worklets/apple/worklets/apple/WorkletsMessageThread.h b/node_modules/react-native-worklets/apple/worklets/apple/WorkletsMessageThread.h
index 822312c..2dce1ed 100644
--- a/node_modules/react-native-worklets/apple/worklets/apple/WorkletsMessageThread.h
+++ b/node_modules/react-native-worklets/apple/worklets/apple/WorkletsMessageThread.h
@@ -1,7 +1,13 @@
 #pragma once
 
 #import <Foundation/Foundation.h>
+#if __has_include(<React/RCTMessageThread.h>)
 #import <React/RCTMessageThread.h>
+#elif __has_include(<React_Core/RCTMessageThread.h>)
+#import <React_Core/RCTMessageThread.h>
+#else
+#error "Cannot find RCTMessageThread.h"
+#endif
 
 namespace facebook {
 namespace react {
